components/gc-featured-link/gc-featured-link-fr.html

```diff
@@ -1,82 +1,71 @@
 ---
-title: Lien en vedette
-description: Documentation et exemple pratique de lien en vedette - fonctionnalité provisoire
-language: fr
-altLangPage: gc-featured-link-en.html
-dateModified: 2021-07-07
-layout: no-container
+{
+	"title": Lien en vedette,
+	"description": Exemples pratiques de la composante Lien en vedette,
+	"language": fr,
+	"altLangPage": gc-featured-link-en.html,
+	"dateModified": 2024-03-08,
+	"breadcrumbs": [
+		{ "title": "Lien en vedette - Documentation", "link": "components/gc-featured-link/gc-featured-link-doc-fr.html" }
+	],
+	"layout": no-container
+}
 ---
+<span class="wb-prettify all-pre hide"></span>
+
 <div class="container">
-	<h1>{{ page.title }}</h1>
-
-	{% include alert-provisional.html %}
-
-	<span class="wb-prettify all-pre hide"></span>
-
-	<dl class="horizontal">
-		<dt>Statut</dt>
-		<dd>Provisoire</dd>
-		<dt>Type</dt>
-		<dd>Conception de Canada.ca</dd>
-		<dt>Dernière revue</dt>
-		<dd>2021-07-07</dd>
-		<dt>Est conforme à</dt>
-		<dd>Spécifications du contenu et de l'architecture de l'information 2.1.2</dd>
-		<dt>Directives</dt>
-		<dd>n/a</dd>
-		<dt>Nom technique</dt>
-		<dd><code>gc-featured-link</code></dd>
-	</dl>
-
-	<h2>Objectif</h2>
-	<p>Ceci ajoute un lien en vedette court et descriptif vers une page que votre institution doit mettre en évidence. Il doit être utilisé pour attirer l'attention sur les avertissements ou les avis.</p>
-
-	<h2>Exemples pratiques</h2>
-
-	<h3>Lien en vedette (Provisiore v1.0)</h3>
-	<p>La structure sémantique logique des en-têtes n'a pas été suivie afin de s'adapter à l'exemple pratique suivant. Vous pouvez aller directement au <a href="#codeSample1">code source</a>.</p>
+	<h1 id="wb-cont" property="name">{{ page.title }}</h1>
+	<p class="mrgn-bttm-lg">Vous trouverez ci-dessous des exemples pratiques pour la composante « Lien en vedette ».</p>
+
+	<section class="alert alert-info">
+		<h2 class="h3">Structure des en-têtes</h2>
+		<p>La structure sémantique logique des en-têtes n'a pas été suivie afin de s'adapter aux exemples pratiques suivants.</p>
+	</section>
+
+	<h2>On this page</h2>
```

Comment:
```suggestion
	<h2>Sur cette page</h2>
```

I think that is the right translation, see the Canada.ca design system for the right translation for this component.